os/net/mac/mac.h

```diff
@@ -43,6 +43,14 @@
 #include "contiki.h"
 #include "dev/radio.h"
 
+/**
+ *\brief The default channel for IEEE 802.15.4 networks.
+ */
+#ifdef IEEE802154_CONF_DEFAULT_CHANNEL
+#define IEEE802154_DEFAULT_CHANNEL           IEEE802154_CONF_DEFAULT_CHANNEL
+#else /* IEEE802154_CONF_DEFAULT_CHANNEL */
+#define IEEE802154_DEFAULT_CHANNEL           26
+#endif /* IEEE802154_CONF_DEFAULT_CHANNEL */
```

Comment:
It's not immediately clear to me why we need this here as well as in `os/contiki-default-conf.h`?

Reply:
Are you suggesting keeping only the `_CONF` version of the flag everywhere, or rather having `contiki-default-conf.h` define the non-`_CONF` version? (currently if only defines `_CONF` flags)

Reply:
Never mind, all good